app/controllers/description_change_validation_requests_controller.rb

```diff
@@ -1,37 +1,44 @@
 # frozen_string_literal: true
 
 class DescriptionChangeValidationRequestsController < ValidationRequestsController
-  before_action :set_planning_application, only: %i[new create]
+  before_action :set_planning_application, only: %i[new create show cancel]
 
   def new
-    @description_change_validation_request = @planning_application.description_change_validation_requests.new
+    @description_change_request = @planning_application.description_change_validation_requests.new
   end
 
   def create
-    @description_change_validation_request = @planning_application.description_change_validation_requests.new(description_change_validation_request_params)
-    @description_change_validation_request.user = current_user
+    @description_change_request = @planning_application.description_change_validation_requests.new(description_change_validation_request_params)
+    @description_change_request.user = current_user
     @current_local_authority = current_local_authority
 
-    if @description_change_validation_request.save
-      email_and_timestamp(@description_change_validation_request) if @planning_application.invalidated?
-
-      flash[:notice] = "Validation request for description successfully created."
-      if @planning_application.invalidated?
-        audit("description_change_validation_request_sent", description_audit_item(@description_change_validation_request, @planning_application),
-              @description_change_validation_request.sequence)
-      else
-        audit("description_change_validation_request_added", description_audit_item(@description_change_validation_request, @planning_application),
-              @description_change_validation_request.sequence)
-      end
-      redirect_to planning_application_validation_requests_path(@planning_application)
-
+    if @description_change_request.save
+      email_and_timestamp(@description_change_request)
+      flash[:notice] = "Description change request successfully sent."
+      audit("description_change_validation_request_sent", description_audit_item(@description_change_request, @planning_application),
+            @description_change_request.sequence)
+      redirect_to planning_application_path(@planning_application)
     else
       render :new
     end
   end
 
+  def show
+    @description_change_request = description_change_request
+  end
+
+  def cancel
+    description_change_request.update!(state: "closed", approved: false, rejection_reason: "Request cancelled by planning officer.")
+    flash[:notice] = "Description change request successfully cancelled."
+    redirect_to planning_application_path(@planning_application)
+  end
+
   private
 
+  def description_change_request
```

Comment:
can you call this `set_description_change_request` and set the instance variable here?